TinyInsights/ApplicationInsightsProvider.cs

```diff
@@ -425,20 +488,25 @@ public async Task TrackPageViewAsync(string viewName, Dictionary<string, string>
     {
         try
         {
-            if (Client is null)
+            if(Client is null)
+            {
+                return;
+            }
+
+            if(!IsTrackPageViewsEnabled)
```

Comment:
This check is missing, so regardless what happens to this PR, this should get added